cognite/client/_api/extractionpipelineruns.py

```diff
@@ -0,0 +1,84 @@
+from typing import *
+
+from cognite.client import utils
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes import ExtractionPipelineRunFilter, ExtractionPipelineRunList
+from cognite.client.data_classes.extractionpipelineruns import ExtractionPipelineRun, StringFilter
+from cognite.client.data_classes.shared import TimestampRange
+
+
+class ExtractionPipelineRunsAPI(APIClient):
```

Comment:
would it make sense to put this class in `extractionpipelines.py`?

Reply:
I'm very open to moving this over, I had it separate because of the two api clients

Reply:
going through the api clients, raw.py and iam.py have their classes in the same file, so would be better to move this one too?

Reply:
Ah, nice catch on consistency there. I'll move it! 🙂